src/binary/merkle_tree.rs

```diff
@@ -90,6 +88,23 @@ where
     // PRIVATE
     //
 
+    fn root_node(&mut self) -> Result<Option<Node>, Box<dyn std::error::Error>> {
```

Comment:
This function is simply an extraction of the existing code at https://github.com/FuelLabs/fuel-merkle/pull/61/files#diff-e809f5638584af006d5dfc8ded1ccc5900cdbeb5b45ba4afc7afbed81dee4ed1L37